### PR TITLE
feat(composer): add composer.json

### PR DESCRIPTION
**Pull request by hypeJunction**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"name": "juho-jaakkola/elgg-tour",
+	"type": "elgg-plugin",
+	"description": "Manage and display feature tours",
+	"homepage": "http://hypejunction.com",
```
Comment (juho-jaakkola) [inline composer.json:5]: ? :)

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		"plugin",
+		"tour",
+		"joyride",
+		"hopscotch"
+	],
+	"license": "GPL-2.0",
+	"authors": [
+		{
+			"name": "Juho Jaakkola",
+			"email": ""
```
Comment (juho-jaakkola) [inline composer.json:17]: juho@elgg.org

File: composer.json (hunk 1 above)
Comment (hypeJunction) [inline composer.json:5]: Whopps copy paste at night...

**Comment by juho-jaakkola** [comment]
Replaced by da6ac824b750d29c00f2d2b0a9397782ba65d84c

**Pull request closed without merging**